### PR TITLE
Remove travis-ci, split README/CONTRIBUTING

### PR DESCRIPTION
**Pull request by bollwyvl**
So long, travis, thanks for all the builds!

This PR:
- [x] removes the travis build config
- [x] splits CONTRIBUTING and README
  - [ ] README
    - [x] adds link to LICENSE  
    - [x] add badges for PyPI and conda-forge
    - [x] add new ci badges
    - [x] add binder badge 
  - [ ] CONTRIBUTING
    - [x] adds Code of Conduct link
    - [x] expand with more development tasks
- [x] binder ([preview](https://mybinder.org/v2/gh/bollwyvl/pythreejs/remove-travis-ci?urlpath=lab%2Ftree%2Fexamples%2FExamples.ipynb))
  - [x] move `environment.yml` to `.binder` folder
  - [x] add rest of (declared) example dependencies 
  - [x] make it a "hot" binder, with current in-repo stuff
  - [x] build docs in binder 
- [x] add README to package so pypi isn't so empty
  - ... as a number of files it links to don't exist on `master`, the docs build will presently fail
    - ... but maybe would _anyway_ because of #351/#352
- [ ] update license years/holders
  - [ ] @jasongrout as the current copyright holder, how do you feel about normalizing the license text, etc. with what's in `ipywidgets`

Not on this PR:
- updating the sphinx install docs
  - maybe pull in the README and CONTRIBUTING in, rather than reproducing it in `rst`

### CODE REVIEW

File: .binder/postBuild [hunk 1]
```diff
@@ -3,17 +3,10 @@ set -eux
...
 
-python -m pip install -vv -e . --no-deps --ignore-installed
-python -m pip check
-
-# install labextension
-pushd js
-jupyter labextension develop . --overwrite
-popd
+python setup.py sdist
+python -m pip install -v dist/pythreejs*.tar.gz --no-deps --ignore-installed
```
Comment (vidartf) [inline .binder/postBuild]: Does this not cause a new build when installing from the sdist? I might be more optimal to install via wheel?

Comment (bollwyvl) [reply .binder/postBuild]: it probably does... i'm more interested in _robust_ than _optimal_, and frankly care more about what makes it to the (intermediate) `.tar.gz` than the `.whl`... but faster is of course better. will try a few more things...

Comment (bollwyvl) [reply .binder/postBuild]: `--no-build-isolation` saves a some stuff (e.g recompiling `pyrsistent` and `pandocfilters`).

Unfortunately, the way `--skip-npm` is passed down, if you have node, it's going to rebuild... perhaps optionally checking a `PYTHREEJS_SKIP_NPM` would be more robust. Or if the `stale` check worked...

**Comment by bollwyvl** [comment]
I think this is about as good as I can get the simple docs and binder working for now. Probably some more issues are needed, but this would probably generate some pretty good packages at this point, module updating the LICENSE information.

File: js/README.md [hunk 2]
```diff
@@ -1,17 +1,62 @@
...
 ```bash
 npm install --save jupyter-threejs
 ```
+
+`yarn`, or the JupyterLab-bundled version `jlpm`, may be also be used.
+
+```bash
+yarn add jupyter-threejs
+jlpm add jupyter-threejs
+```
```
Comment (vidartf) [inline js/README.md:58]: The `npm/yarn/jlpm add` command is not a dev install prerequisite. I believe the previous bit here was also not very helpful, so it is probably best to delete all of it and just link to the contributing document.

Comment (bollwyvl) [reply js/README.md:58]: cool.

Comment (bollwyvl) [reply js/README.md]: Tuned that up, and added the LICENSE so that it gets included in the npm distribution (and all the other places).

Comment (bollwyvl) [reply js/README.md]: also, of note: there's no `.npmignore` or `files` in package JSON, so... a _lot_ of stuff gets in there (scripts, src, the linter config, the lab dist tarball, etc). that could probably be tuned up...

Comment (vidartf) [reply js/README.md]: Good point, I'll have a look at cleaning up the tarball.

File: .binder/postBuild (hunk 1 above)
Comment (vidartf) [reply .binder/postBuild]: To be clear, my suggestion was to use `python setup.py bdist_wheel` and then pip install using the wheel. That should do what you want while still avoiding rebuilds, right ?

Comment (vidartf) [reply .binder/postBuild]: (if you are saying this is meant to be a test of the sdist install, that is probably different).

**Comment by vidartf** [comment]
Merging as is, will iterate on the tarball and then do the release.

**Comment by bollwyvl** [comment]
Gave the binder a shake, looking good. 

Now in badgetopia, we're just down to docs... will have a look...

**Pull request merged**